core/domain/skill_services_test.py

```diff
@@ -307,6 +309,193 @@ def test_get_skill_summary_by_id(self):
         self.assertEqual(skill_summary.description, 'Description')
         self.assertEqual(skill_summary.misconception_count, 1)
 
+    def test_get_all_skill_summary_dicts_with_topic_and_classroom(self):
+        skill_summaries_dicts = (
+            skill_services.get_all_skill_summary_dicts_with_topic_and_classroom()) # pylint: disable=line-too-long
```

Comment:
Please revise this PR so that there are no more pylint-disables in any file.

Reply:
Done.